### PR TITLE
Super admin permissions note

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Super admin permissions note
- **Is this PR related to a Monolith release?** Yes 2024.02.0

Netlify Link:
To bullet added in "What you need" section - https://65bc06f8b9fae3077df2887e--reverent-murdock-829d24.netlify.app/docs/guides/configure-direct-auth-grants/aotp/main/

Note added in each "Set up your app" section - https://65bc06f8b9fae3077df2887e--reverent-murdock-829d24.netlify.app/docs/guides/configure-direct-auth-grants/aotp/main/#set-up-your-app

### Resolves:

* [OKTA-690693](https://oktainc.atlassian.net/browse/OKTA-690693)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/aotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Faotp%2Fnutrition.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2e33a561-436d-4ac8-a586-a5d1b465033c) | 100  | 35  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/bmfaotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fbmfaotp%2Fnutrition.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3587a149-e580-40a9-b943-b513deb93bbd) | 100  | 36  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fcoobov%2Fnutrition.md) | [80](https://okta.acrolinx.cloud/api/v1/checking/scorecards/792bb50a-453c-48d0-978b-fcde0a84b251) | 100  | 37  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/dmfaoobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fdmfaoobov%2Fnutrition.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/75fdd874-f449-4ee5-85a8-4afe4c9c157d) | 100  | 38  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/eoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Feoobsv%2Fnutrition.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a1f25e67-cbfc-4c8b-9d9e-fa836c396408) | 100  | 38  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/fmfaoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Ffmfaoobsv%2Fnutrition.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bec4334c-50a4-4dad-b7d0-adbec5813f06) | 100  | 40  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/index.md](https://github.com/okta/okta-developer-docs/blob/78fc183674751f531380c780ff16acd5424b8713/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/cda0412f-c8cb-4318-bde3-af2d8721e89b) | 88  | 88  | 65  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.ebaf59c8-1fd4-4a0d-b067-c00fc3599158?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/aotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Faotp%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4f76065c-57c7-4bff-b301-600b3177a78e) | 100  | 45  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/bmfaotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fbmfaotp%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d2985ef9-7c5a-4705-bd15-26699a42f633) | 100  | 46  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fcoobov%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e6da5b88-5d6d-4c54-a5a4-e4f800f68602) | 100  | 46  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/dmfaoobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fdmfaoobov%2Fnutrition.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8b725df2-6d16-423d-ac79-21bf314fe889) | 100  | 48  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/eoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Feoobsv%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ae8a0610-2c83-4235-98fd-cae9d66a2e82) | 100  | 47  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/fmfaoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Ffmfaoobsv%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6c4c4841-1892-48d7-af71-2c9e9f2d79f3) | 100  | 49  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/index.md](https://github.com/okta/okta-developer-docs/blob/f6d82c0acf4a14dc4431bcf686cddd54cde45709/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0931f06c-f3f6-4a56-adbb-ad907ae3149d) | 88  | 88  | 65  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.36abed04-d679-4c3e-b76c-edc30b95f7bb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/nutrition.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <ApiLifecycle access="ea" />
 <ApiLifecycle access="ie" />
 
-Use this guide to implement a direct authentication out-of-band (OOB) flow for your app in Okta. The guide focuses on using Okta Verify Push with the Okta Verify authenticator.
+Use this guide to implement a direct authentication out-of-band (OOB) flow for your app in Okta. The guide uses Okta Verify Push with the Okta Verify authenticator.
```
Comment (dannavarro-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/nutrition.md:4]: In the below file you also updated `The guide` to `This guide`

File: packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/dmfaoobov/nutrition.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <ApiLifecycle access="ea" />
 <ApiLifecycle access="ie" />
 
-Use this guide to implement a direct authentication multifactor out-of-band (MFA OOB) flow for your app in Okta. The guide focuses on using Okta Verify Push with the Okta Verify authenticator.
+Use this guide to implement a direct authentication multifactor out-of-band (MFA OOB) flow for your app. This guide uses Okta Verify Push with the Okta Verify authenticator.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/dmfaoobov/nutrition.md:4]:
```suggestion
Use this guide to implement a direct authentication multifactor out-of-band (MFA OOB) flow for your app. The guide uses Okta Verify Push with the Okta Verify authenticator.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/aotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Faotp%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b022b60a-3faa-4f2b-9b69-ab1d9423e3cf) | 100  | 45  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/bmfaotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fbmfaotp%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c0bc7b7e-2eee-4ae7-a05e-ae423ed334ae) | 100  | 46  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fcoobov%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0d0b5e72-9b5b-4c6f-934f-5ede2b518d30) | 100  | 46  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/dmfaoobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fdmfaoobov%2Fnutrition.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ed1e5656-438d-4921-86fa-1b2cfb13a2c7) | 100  | 48  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/eoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Feoobsv%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fd9d356e-c3e6-4689-837c-5ede22fe2308) | 100  | 47  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/fmfaoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Ffmfaoobsv%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b5340ccd-c45c-4fd2-951e-e7b2505d92e6) | 100  | 49  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/index.md](https://github.com/okta/okta-developer-docs/blob/6770f9fc1bfe227c3e296fc098bf6786ac13f50b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/97e228cc-90a6-4364-8dac-310269cdab7a) | 88  | 88  | 65  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.9a6531f0-f3b2-47b6-9d61-cd5048ebfa76?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/aotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Faotp%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e4818fe6-33d1-4c46-9f09-970d2822cd2d) | 100  | 45  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/bmfaotp/nutrition.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fbmfaotp%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/45d37f88-1f98-4fe3-91ab-db5e1e2352ed) | 100  | 46  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/coobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fcoobov%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/37ddac69-23bc-40f0-b5f8-20384299e214) | 100  | 46  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/dmfaoobov/nutrition.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Fdmfaoobov%2Fnutrition.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/20c9c58d-6112-41b6-af24-eb711bd2462c) | 100  | 48  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/eoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Feoobsv%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4c79acd0-0770-47df-88ca-4ab0018c3ecd) | 100  | 47  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/fmfaoobsv/nutrition.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Ffmfaoobsv%2Fnutrition.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a92693d8-eeac-4955-92e0-57cbdc87e1c3) | 100  | 49  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/index.md](https://github.com/okta/okta-developer-docs/blob/2e7076f87c487bb61043ff6e6c9a44386bf8b031/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fconfigure-direct-auth-grants%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/af18518f-feff-4061-9667-5699a77d4238) | 88  | 88  | 65  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.3ffeecab-42c4-4f0c-a554-145be87bf3d1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by dannavarro-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/fmfaoobsv/nutrition.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <ApiLifecycle access="ea" />
 <ApiLifecycle access="ie" />
 
-This guide explains how to implement a direct authentication multifactor out-of-band (MFA OOB) flow for your app in Okta. The guide focuses on using the Phone authenticator with the SMS or Voice factor.
+This guide explains how to implement a direct authentication multifactor out-of-band (MFA OOB) flow for your app. The guide uses the Phone authenticator with the SMS or Voice factor.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/fmfaoobsv/nutrition.md:4]:
```suggestion
This guide explains how to implement a direct authentication multifactor out-of-band (MFA OOB) flow for your app. The guide uses the phone authenticator with the SMS or voice factor.
```

File: packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/aotp/nutrition.md
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/aotp/nutrition.md:1]: You could change this to "When you create or update an app..." for simplicity but it's up to you :)

**Review by barbaravo-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/index.md [hunk 4]
```diff
@@ -63,6 +63,8 @@ If your Okta org uses Identity Engine, then the <StackSnippet snippet="granttype
 
 Before you can implement authorization, you need to register your app in Okta by creating an app integration from the Admin Console.
 
+> **Note:** When you create an app, or update an existing app, you must have super admin permissions to enable direct authentication grant types.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-direct-auth-grants/main/index.md:66]:
```suggestion
> **Note:** When you create or update an app, you must have super admin permissions to enable direct authentication grant types.
```

**Pull request merged**